backend/ddang/src/main/java/com/ddang/ddang/chat/domain/ChatRoom.java

```diff
@@ -47,19 +47,32 @@ public ChatRoom(final Auction auction, final User buyer) {
     }
 
     // TODO : 쪽지 비활성화 일수 리팩토링시 수정
-    public boolean isChatAvailable(final LocalDateTime targetTime) {
+    public boolean isChatAvailableTime(final LocalDateTime targetTime) {
         final LocalDateTime maxChatTime = getCreatedTime().plusDays(CHAT_EXPIRATION_DAY);
 
         return targetTime.isBefore(maxChatTime);
     }
 
     public User calculateChatPartnerOf(final User user) {
-        final User seller = auction.getSeller();
-
-        if (user.equals(seller)) {
+        if (isSeller(user)) {
             return buyer;
         }
 
-        return seller;
+        return auction.getSeller();
+    }
+
+    public boolean isParticipant(final User user) {
+        if (isSeller(user)) {
+            return true;
+        }
+        return isBuyer(user);
```

Comment:
```suggestion
        return isBuyer(user) || isSeller(user)
```

이런식으로 줄일 수 있지 않을까요?!

Reply:
그러네요! 왜 이렇게 적었지? 레벨 1 다시 해야겠네요 ㅠㅠ